pkg/kubelet/dockertools/labels.go

```diff
@@ -73,6 +73,10 @@ func GetPodName(info *cadvisorapiv2.ContainerInfo) string {
 	return info.Spec.Labels[kubernetesPodNameLabel]
 }
 
+func GetPodUID(info *cadvisorapiv2.ContainerInfo) string {
```

Comment:
We should not inject cadvisor API into docker package :(. Not for this PR. but we should go back and clean this up.